### PR TITLE
Make sure payment request print view doesn't show table header twice

### LINKED ISSUE #4341: [Bug]: UI Print View on Payment Requests
### What is your BTCPay version?

v.1.7.x.x Altcoins (integration branch from dennis)

### How did you deploy BTCPay Server?

Docker

### What happened?

When printing a succeeded payment request, I got the following in the printing preview:
image

### How did you encounter this bug?

I created a payment request, paid it and opened the "print" view. Microsoft Edge opened the print preview window and the above came to show on a Lightning payed payment request.

### Relevant log output

_No response_

### What browser do you use?

Microsoft Edge on MacOS

### Additional information

_No response_

### Are you sure this is a bug report?

- [X] I confirm this is a bug report

### PR DESCRIPTION
**Pull request by bolatovumar**
|Before|After|
|---|---|
|before|after|

close #4341

### CODE REVIEW

File: BTCPayServer/Views/UIPaymentRequest/ViewPaymentRequest.cshtml [hunk 1]
```diff
@@ -350,7 +355,7 @@
                                             </tr>
                                             <template v-for="payment of invoice.payments">
                                                 <tr class="table-borderless table-light">
-                                                    <td class="text-break"><code>{{payment.destination}}</code></td>
+                                                    <td class="text-break"><code>bcrt1qa566r3y3ngwxrxl0wr97pp9q8kl5zpkv0u377zbcrt1qa566r3y3ngwxrxl0wr97pp9q8kl5zpkv0u377zbcrt1qa566r3y3ngwxrxl0wr97pp9q8kl5zpkv0u377zbcrt1qa566r3y3ngwxrxl0wr97pp9q8kl5zpkv0u377zbcrt1qa566r3y3ngwxrxl0wr97pp9q8kl5zpkv0u377zbcrt1qa566r3y3ngwxrxl0wr97pp9q8kl5zpkv0u377zbcrt1qa566r3y3ngwxrxl0wr97pp9q8kl5zpkv0u377zbcrt1qa566r3y3ngwxrxl0wr97pp9q8kl5zpkv0u377zbcrt1qa566r3y3ngwxrxl0wr97pp9q8kl5zpkv0u377zbcrt1qa566r3y3ngwxrxl0wr97pp9q8kl5zpkv0u377zbcrt1qa566r3y3ngwxrxl0wr97pp9q8kl5zpkv0u377zbcrt1qa566r3y3ngwxrxl0wr97pp9q8kl5zpkv0u377zbcrt1qa566r3y3ngwxrxl0wr97pp9q8kl5zpkv0u377zbcrt1qa566r3y3ngwxrxl0wr97pp9q8kl5zpkv0u377z</code></td>
```
Comment (dennisreimann) [inline BTCPayServer/Views/UIPaymentRequest/ViewPaymentRequest.cshtml:358]: Can you revert this? Looks like it's a remainder from testing.

**Review by dennisreimann — CHANGES_REQUESTED** [review]
Might make sense to also add this in general to the site.cas

File: BTCPayServer/Views/UIPaymentRequest/ViewPaymentRequest.cshtml (hunk 1 above)
Comment (dennisreimann) [reply BTCPayServer/Views/UIPaymentRequest/ViewPaymentRequest.cshtml:358]:
```suggestion
                                                                                                        <td class="text-break"><code>{{payment.destination}}</code></td>
```

File: BTCPayServer/Views/UIPaymentRequest/ViewPaymentRequest.cshtml [hunk 2]
```diff
@@ -350,7 +355,7 @@
                                             </tr>
                                             <template v-for="payment of invoice.payments">
                                                 <tr class="table-borderless table-light">
-                                                    <td class="text-break"><code>{{payment.destination}}</code></td>
+                                                                                                        <td class="text-break"><code>{{payment.destination}}</code></td>
```
Comment (dennisreimann) [inline BTCPayServer/Views/UIPaymentRequest/ViewPaymentRequest.cshtml:358]:
```suggestion
                                                    <td class="text-break"><code>{{payment.destination}}</code></td>
```

**Review by dennisreimann — APPROVED** [review]

**Review by dstrukt — APPROVED** [review]
LGTM 🏆

**Pull request merged**